src/modules/commander/PreflightCheck.cpp

```diff
@@ -562,15 +563,15 @@ static bool ekf2Check(orb_advert_t *mavlink_log_pub, bool optional, bool report_
 	}
 
 out:
+	orb_unsubscribe(sub);
 	return success;
 }
 
-bool preflightCheck(orb_advert_t *mavlink_log_pub, bool checkMag, bool checkAcc, bool checkGyro,
-		    bool checkBaro, bool checkAirspeed, bool checkRC, bool checkGNSS,
+bool preflightCheck(orb_advert_t *mavlink_log_pub, bool checkSensors, bool checkAirspeed, bool checkRC, bool checkGNSS,
 		    bool checkDynamic, bool isVTOL, bool reportFailures, bool prearm, hrt_abstime time_since_boot)
 {
 
-	if (time_since_boot < 1e6) {
+	if (time_since_boot < 2000000) {
```

Comment:
1. In the PR it says something about 20 seconds, but `2000000` is actually 2 seconds (2.000.000). 
2. Shouldn't this check go into the `airspeedCheck()` function?